_stdlib/code/_macros.dm

```diff
@@ -65,7 +65,7 @@ var/list/global_spawn_dbg = list()
 
 // Why the separate mask check? NPCs don't use assigned_role and we still wanna play the cluwne-specific sound effects.
 #define iscluwne(x) ((x?.mind?.assigned_role == "Cluwne") || istype(x.wear_mask, /obj/item/clothing/mask/cursedclown_hat))
-#define ishorse(x) (istype(x, /mob/living/carbon/human) && ((x:mind && x:mind.assigned_role && x:mind:assigned_role == "Horse") || istype(x:wear_mask, /obj/item/clothing/mask/horse_mask/cursed)))
+#define ishorse(x) (istype(x, /mob/living/carbon/human) && ((x.mind?.assigned_role == "Horse") || istype(x.wear_mask, /obj/item/clothing/mask/horse_mask/cursed)))
```

Comment:
In theory I fear we might call this on non-humans, just like `iscluwne` in which case it won't compile but we're not doing that now and honestly I'm too scared to look for the piece of code that calls `ishorse`.